docs/sources/explore/logs-integration.md

```diff
@@ -14,113 +14,112 @@ weight: 15
+- **Total bytes processed**: Represents the cumulative size of the log data processed in bytes.
 
-### Prettify JSON
+> **Note:** The availability of certain meta information may depend on the data source, and as a result, you may only see some of these details for specific data sources.
 
-Set this to `true` to pretty print all JSON logs. This setting does not affect logs in any format other than JSON.
+### Escaping newlines
 
-### Deduplication
+Explore automatically detects some incorrectly escaped sequences in log lines, such as newlines (`\n`, `\r`) or tabs (`\t`). When it detects such sequences, Explore provides an "Escape newlines" option.
 
-Log data can be very repetitive and Explore can help by hiding duplicate log lines. There are a few different deduplication algorithms that you can use:
+To automatically fix incorrectly escaped sequences that Explore has detected:
 
-- **Exact -** Exact matches are done on the whole line except for date fields.
-- **Numbers -** Matches on the line after stripping out numbers such as durations, IP addresses, and so on.
-- **Signature -** The most aggressive deduplication, this strips all letters and numbers and matches on the remaining whitespace and punctuation.
+1. Click "Escape newlines" to replace the sequences.
+2. Manually review the replacements to confirm their correctness.
 
-### Display results order
+Explore replaces these sequences. When it does so, the option will change from "Escape newlines" to "Remove escaping". Evaluate the changes as the parsing may not be accurate based on the input received. You can revert the replacements by clicking "Remove escaping".
 
-You can change the order of received logs from the default descending order (newest first) to ascending order (oldest first).
+### Log level
 
-## Labels and detected fields
+For the logs where a `level` label is specified, we use the value of the label to determine the log level and update color of log line accordingly. If the log doesn't have a level label specified, we try to find out if its content matches any of the supported expressions (see below for more information). The log level is always determined by the first match. In case Grafana is not able to determine a log level, it will be visualized with an unknown log level.
 
-Each log line has an extendable area, with its labels and detected fields, for more robust interaction. You can filter for (positive filter) and filter out (negative filter) selected labels. Additionally, you can select a unique field to visualize instead of the whole log line by clicking on the eye icon.
-Finally, each field or label also has a stats icon to display ad-hoc statistics in relation to all displayed logs.
+> **Tip:** If you use Loki data source and the "level" is in your log-line, use parsers (JSON, logfmt, regex,..) to extract the level information into a level label that is used to determine log level. This will allow the histogram to show the various log levels in separate bars.
```

Comment:
```suggestion
> **Tip:** If you use a Loki data source and the "level" is part of your log line, you can use parsers (JSON, logfmt, regex,..) to extract the level information into a level label that is used to determine the level value. This will allow the histogram to show the various log levels as separate bars.
```